cmd/vclusterctl/cmd/platform/share/space.go

```diff
@@ -31,24 +31,24 @@ type SpaceCmd struct {
 	Log log.Logger
 }
 
-// NewSpaceCmd creates a new command
-func NewSpaceCmd(globalFlags *flags.GlobalFlags, defaults *pdefaults.Defaults) *cobra.Command {
-	cmd := &SpaceCmd{
+// NewNamespaceCmd creates a new command
+func NewNamespaceCmd(globalFlags *flags.GlobalFlags, defaults *pdefaults.Defaults) *cobra.Command {
+	cmd := &NamespaceCmd{
 		GlobalFlags: globalFlags,
 		Log:         log.GetInstance(),
 	}
-	description := product.ReplaceWithHeader("share space", `
-Shares a space with another vCluster platform user or team. The user
+	description := product.ReplaceWithHeader("share namespace", `
+Shares a vCluster platform namespace with another platform user or team. The user
 or team need to have access to the cluster.
 Example:
-vcluster platform share space myspace
-vcluster platform share space myspace --project myproject
-vcluster platform share space myspace --project myproject --user admin
+vcluster platform share namespace myspace
+vcluster platform share namespace myspace --project myproject
+vcluster platform share namespace myspace --project myproject --user admin
 ########################################################
 	`)
 	c := &cobra.Command{
-		Use:   "space" + util.SpaceNameOnlyUseLine,
-		Short: product.Replace("Shares a space with another vCluster platform user or team"),
+		Use:   "namespace" + util.SpaceNameOnlyUseLine,
+		Short: product.Replace("Shares a vCluster platform namespace with another platform user or team"),
 		Long:  description,
 		Args:  util.SpaceNameOnlyValidator,
```

Comment:
Should probably also rename `SpaceNameOnlyValidator` to `NamespaceNameOnlyValidator`